### DISCUSSION TITLE
请问这个输入法键盘的布局生成文件是哪个呢？

### DISCUSSION THREAD

**Discussion by keoned** [fcitx5-android/fcitx5-android#524] [Q&A | 问答]
这个输入法键盘的布局具体是哪个文件生成的呢？

**Comment by rocka** [comment]
https://github.com/fcitx5-android/fcitx5-android/blob/1865b15fe46cadd1265b5b8d400ed11b64504d05/app/src/main/java/org/fcitx/fcitx5/android/input/keyboard/TextKeyboard.kt#L31-L75

**Reply by keoned** [reply]
你好，我还想请教一下您，我现在要实现一个电视遥控器焦点的功能，也是在这里实现吧，可不可以给个大概的思路呢？谢谢
